src/main/java/edu/greenblitz/robotName/subsystems/shooter/Funnel/NeoFunnel/NeoFunnel.java

```diff
@@ -0,0 +1,42 @@
+package edu.greenblitz.robotName.subsystems.shooter.Funnel.NeoFunnel;
+
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.greenblitz.robotName.subsystems.shooter.Funnel.FunnelInputsAutoLogged;
+import edu.greenblitz.robotName.subsystems.shooter.Funnel.IFunnel;
+import edu.greenblitz.robotName.utils.motors.GBSparkMax;
+import edu.wpi.first.math.filter.Debouncer;
+
+import static edu.greenblitz.robotName.subsystems.shooter.Funnel.NeoFunnel.NeoFunnelConstants.*;
+
+public class NeoFunnel implements IFunnel {
+	
+	private GBSparkMax motor;
```

Comment:
Why is it a NEO? do we need a lot of power here? A brushed motor will have less problems, work better and give the same result.
Wait until @NoamRosenberg08 Talks to mechanics but probably we will switch so Brushed

Reply:
when we know what will be the motor ill change